### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RadchenkoVlada**

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<!-- We should put HTML that we want to be visible on every page of our site, such as headers, footers, copyright notices, meta tags, and so on, in the base template. Then, any template inheriting from it will automatically get all that common HTML included automatically, and we will only need to override the HTML code for the block we want to customize. You'll see this principal of creating and overriding blocks in base templates used throughout the projects in this book.
```
Comment (w-koder) [inline templates/base.html]: Please, read the comment and implement your project structure according to it. Please, separate code that should be used for all pages and for main page only. Leave generic code in base.html and code that is used for main page in separate file main.html. I suppose, Django features should be use to accomplish it, not only HTML

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+*.
+*~
+*.pyc
+__pycache__/
+.env
+.venv
+env/
+myvenv/
+.DS_Store
+*~
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply .gitignore]: Done

File: mysite/settings.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'x%a+m1#fez5)&$_%19)v$h9#24#&969!o@_pw8m)r7g!#wq4g('
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply mysite/settings.py]: Done
# SECURITY WARNING: don't run with debug turned on in production!
DEBUG = False

File: mysite/settings.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply mysite/settings.py]: Done
ALLOWED_HOSTS = ['127.0.0.1']

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+Django~=2.2.4
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply requirements.txt]: Screen Shot 2019-10-15 at 21 36 06
higher proof,
now i have 2.2.5 django

File: templates/base.html [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+    <meta charset="utf-8">
+    <meta name="author" content="Radchenko Vlada">
+    <meta name="description" content="RentalCars home page website template"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="static/css/main.css">
+    <title>Font connection</title>
+    <link href="https://fonts.googleapis.com/css?family=Muli|Noto+Sans+JP&display=swap" rel="stylesheet">
+    <!-- in css file
+    font-family: 'Muli', sans-serif;
+    font-family: 'Noto Sans JP', sans-serif; -->
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply templates/base.html]: no

File: mysite/urls.py [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+from django.conf.urls import url
+from django.contrib import admin
+from django.views.generic import TemplateView
+from rentalcar.views import UserRegistrationView
+
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    url(r'^$', TemplateView.as_view(template_name='base.html'), name='home'),
+    url(r'^new-user/$', UserRegistrationView.as_view(), name='user_registration'),
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply mysite/urls.py]: It is used for a new page 'user_registration.html'

File: rentalcar/views.py [hunk 8]
```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import render
+
+# Create your views here.
+from django.contrib.auth.forms import UserCreationForm
+from django.urls import reverse
+from django.views.generic import CreateView
+
+
+class UserRegistrationView(CreateView):
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply rentalcar/views.py]: It hadn't been used, so I deleted it

File: templates/base.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply templates/base.html]: Done, i have created new - header_footer.html. There are Html classes "my_header" and "footer". At the moment I also have files like home.html, user_registration.html

File: templates/base.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply templates/base.html]: This is a tip for me on how to use other fonts in CSS.

File: templates/base.html [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+    
+  </head>
+  <body>
+    <header class="my_header">
+      <div class="site_title">
+        <ul>
+          <li>
+            <a class="site_title" href="#home">Vlada'sCars </a>
+          </li>
+        </ul>
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply templates/base.html]: Done

File: templates/base.html [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+      </div>
+      <div class="menu">
+        <ul>
+          <li class="dropdown">
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply templates/base.html]: Class="dropdown" was renamed to class="sign_in_dropdawn"

File: templates/base.html [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+            <a href="javascript:void(0)" class="dropbtn">Sign in</a>
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply templates/base.html]: Done

File: templates/base.html [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+    <div class="main_content">
+      <div class="search">
+        <!--   If you click "Submit", the form-data will be sent to a page called "/handler.php".
+        TODO: revise implementation and rework according to django, change file.php to file.py-->
+        <form name="SearchResultsForm" method="get"  action="handler.php">
+          <p style="margin:5px">Search for a car</p>
+          <div style="margin:10px"><input style="font-size:20px;" name="pick_up_location" maxlength="30" size="40" placeholder="Type a city or an airport name"></div>
+        </form>
+        <div>
+          <form style="font-size:25px; margin:25px "action="/action_page.php">
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply templates/base.html]: done

File: templates/base.html [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+                <option value="9:00 PM">9:00 PM</option>
+                <option value="10:00 PM">10:00 PM</option>
+                <option value="11:00 PM">11:00 PM</option>
+                <option value="12:00 AM">12:00 AM</option>
+                <option value="1:00 AM">1:00 AM</option>
+                <option value="2:00 AM">2:00 AM</option>
+                <option value="3:00 AM">3:00 AM</option>
+                <option value="4:00 AM">4:00 AM</option>
+                <option value="5:00 AM">5:00 AM</option>
+                <option value="6:00 AM">6:00 AM</option>
```
[the remark this answers is not in the document]
Comment (RadchenkoVlada) [reply templates/base.html]: I have created a new issue

**Pull request closed without merging**